content/post/elm-at-rakuten/index.md

```diff
@@ -347,193 +409,241 @@ This feature is so crucial that it inspired a new coding style: **Compiler Drive
 
 [^the-elm-architecture]: Details about The Elm Architecture can be found in the [official Elm Guide](https://guide.elm-lang.org/architecture/). The Elm Architecture is the predominant way to build applications in Elm. Different variants are also possible. [Elm-spa](https://www.elm-spa.dev/guide) by Ryan Haskell-Glatz is a tool that helps to create single-page applications and create extra abstraction above The Elm Architecture. Rémi Lefèvre built the [RealWorld example app](https://github.com/gothinkster/realworld) using the [Effect pattern](https://discourse.elm-lang.org/t/realworld-example-app-architected-with-the-effect-pattern/5753).
-
 [^unidirectional-data-flow]: The Elm Architecture is based on [**unidirectional data flow**](https://youtu.be/jl1tGiUiTtI?t=470) (a.k.a. **one-way data binding**) like React, in contrast to the **bidirectional data flow** (a.k.a. **two-way data binding**) of frameworks like Angular, Vue, and Svelte ([in Svelte two-way binding can be disabled](https://github.com/sveltejs/svelte/issues/54)). There have been issues with two-way data binding. For example, the many-to-many dependencies between the view and the model can create an infinite loop of cascading updates. Another issue is the lack of control of the change detection mechanism. It is an implicit behavior that is not easy to control. Unidirectional data flow tends to be more predictable.
 
 ![The Elm Architecture, animated](./images/the-elm-architecture.svg)
-*A simple representation of the unidirectional data flows in The Elm Architecture. (Source: The Elm Guide)*[^representation-of-unidirectional-flow]
+_A simple representation of the unidirectional data flows in The Elm Architecture. (Source: The Elm Guide) [^representation-of-unidirectional-flow]_
 
-[^representation-of-unidirectional-flow]: The illustration **A simple representation of the Elm Architecture** is from the as depicted in the [Elm Guide](https://guide.elm-lang.org/architecture/) used under the [CC BY-NC-ND 4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/) [license](https://github.com/evancz/guide.elm-lang.org/blob/master/LICENSE).
+[^representation-of-unidirectional-flow]: The illustration **A simple representation of the Elm Architecture** is from the [Elm Guide](https://guide.elm-lang.org/architecture/) used under the [CC BY-NC-ND 4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/) [license](https://github.com/evancz/guide.elm-lang.org/blob/master/LICENSE).
 
 There are three building blocks in The Elm Architecture:
 
-* `Model` --- the state of your application, the only thing that can mutate
-* `view` --- a way to turn your state into HTML
-* `update` --- a way to update your state based on the `Model` and the *messages*
+- `Model` --- the state of your application, the only thing that can mutate
+- `view` --- a way to turn your state into HTML
+- `update` --- a way to update your state based on the `Model` and the _messages_
```

Comment:
nitpick: should they all be capitalized? Like

```
- `Model`
- `View`
- `Update`
```

Or is there a reason to only capitalize model?

Reply:
Model is a type and in Elm types are capitalized. view and update are functions and functions are not capitalized.

A familiar notation for Elm devs but probably awkward for non-Elm devs.